tests/shell/shell_recorder/shell_recorder.sh

```diff
@@ -28,7 +28,8 @@ else
         KDBCOMMAND="kdb"
 fi
 
-
+shopt -s expand_aliases
```

Comment:
Thanks for the information. I converted the alias to a function it in commit ab772e8.